alacritty.yml

```diff
@@ -497,45 +501,79 @@
 #   - ScrollToTop
 #   - ScrollToBottom
 #   - ClearHistory
+#       Remove the terminal's scrollback history.
 #   - Hide
+#       Hide the Alacritty window.
 #   - Minimize
+#       Minimize the Alacritty window.
 #   - Quit
+#       Quit Alacritty
 #   - ToggleFullscreen
 #   - SpawnNewInstance
+#       Spawn a new instance of Alacritty that preserves the path. Only applies to Linux/BSD.
 #   - ClearLogNotice
+#       Clear Alacritty's UI warning and error notices.
 #   - ClearSelection
+#       Remove the active selection.
 #   - ReceiveChar
 #   - None
 #
 #   (`mode: Vi` only):
 #   - Open
+#       Attempt to open URL under the cursor. See also url.launcher.
```

Comment:
If there is no URL under the cursor, this does nothing. We don't even try to open it as a URL. So we should be clear that this opens the URL if there is one.

```suggestion
#       Opens URLs at the cursor location with the launcher configured in `url.launcher`.
```